x-pack/plugins/alerting/server/rules_client/tests/bulk_delete.test.ts

```diff
@@ -0,0 +1,397 @@
+/*
```

Comment:
Might have missed them but I couldn't find any tests that ensure that the correct audit logs are logged for each operation. This is an important security feature and should be validated.

Reply:
Thanks for feedback, @thomheymann!